### PR TITLE
Create page for adding an Admin

### PR DESCRIPTION
**Pull request by atulyamiglani**
creates the add-admin-form.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +0,0 @@
-AWS_COGNITO_USERNAME=secret_username
```
Comment (onb6) [inline .env.example:1]: pls don't delete this file

File: src/pages/AddAdminPage.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      await apiClient.createUser(values.email, Role.ADMIN);
+      toast({
+        title: 'Post created.',
+        description: `Added the admin with the email ${values.email}.`,
+        status: 'success',
+        duration: 4000,
+        isClosable: true,
+      });
+    } catch (err) {
+      // todo: add alert pop up
```
Comment (rymaju) [inline src/pages/AddAdminPage.tsx:24]:
```suggestion
```

File: src/pages/AddAdminPage.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    values: AdminFormValues,
+    actions: FormikHelpers<Partial<AdminFormValues>>,
+  ): Promise<void> {
+    try {
+      await apiClient.createUser(values.email, Role.ADMIN);
+      toast({
+        title: 'Post created.',
+        description: `Added the admin with the email ${values.email}.`,
+        status: 'success',
+        duration: 4000,
```
Comment (rymaju) [inline src/pages/AddAdminPage.tsx:20]: 4000 is a magic number, please extract it as a constant, e.x `TOAST_POPUP_DURATION =  4000 //ms`

File: src/pages/EditAdminPage.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Grid, Box, Container, Heading, GridItem } from '@chakra-ui/react';
```
Comment (rymaju) [inline src/pages/EditAdminPage.tsx:1]: Delete this file

File: cypress/integration/add-admin.spec.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+            email,
+            role: "admin",
+          }).as("createAdmin");
+        cy.loginAsResearcher();
+        cy.visit('/admin/add-new-admin'); 
+        cy.get('#admin-email').click();
+        cy.get('#admin-email').type(email); 
+        cy.get('#submit-admin-email').click();
+        cy.wait('@createAdmin');
+        cy.contains(`Added the admin with the email ${email}.`);
```
Comment (rymaju) [inline cypress/integration/add-admin.spec.ts:17]: Might need an assertion here.
```suggestion
        cy.contains(`Added the admin with the email ${email}.`).visible();
```

File: src/components/form/AddAdminForm.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (rymaju) [inline src/components/form/AddAdminForm.tsx:1]: 🏕️ We will want to refactor this with my new form abstraction. You don't need to do that now but note that we will not be doing manual formik forms going forward.

File: src/api/apiClient.ts [hunk 7]
```diff
@@ -39,13 +40,22 @@ export class ApiClient {
     return this.axiosInstance.get(path).then((response) => response.data);
   }
 
+  private async post(path: string, body: unknown): Promise<unknown> {
+    const res = await this.axiosInstance.post(path, body);
```
Comment (rymaju) [inline src/api/apiClient.ts:44]: Redundant variable, just return this value

Comment (rymaju) [reply src/api/apiClient.ts:44]: See the `get` method above

File: src/api/apiClient.ts [hunk 8]
```diff
@@ -39,13 +40,22 @@ export class ApiClient {
...
   public async getHello(): Promise<string> {
     return this.get('/') as Promise<string>;
   }
 
   public async getMe(): Promise<User> {
     return this.get('/auth/me') as Promise<User>;
   }
+
+  public async createUser(email: string, role: Role): Promise<User> {
+    return this.post('/user', { email, role }) as Promise<User>;
```
Comment (rymaju) [inline src/api/apiClient.ts:57]: Missing pact test, see @DerekL27 's PR or the existing pact tests for examples

**Review by rymaju — APPROVED** [review]
LGTM

**Review by onb6 — APPROVED** [review]
Woohoo!

**Pull request merged**